pyrealm/pmodel/functions.py

```diff
@@ -194,6 +197,76 @@ def calc_ftemp_inst_vcmax(
     return fva * fvb
 
 
+def calc_modified_arrhenius_factor(
+    tk: NDArray,
+    Ha: float | NDArray,
+    Hd: float | NDArray,
+    deltaS: float | NDArray,
+    mode: str = "M2002",
+    tk_ref: NDArray | None = None,
+    pmodel_const: PModelConst = PModelConst(),
+    core_const: CoreConst = CoreConst(),
+) -> NDArray:
+    r"""Calculate the modified Arrhenius factor with temperature for an enzyme.
+
+    This function returns a temperature-determined factor expressing the rate of an
+    enzymatic process relative to the rate at a given reference temperature. This is
+    used in the calculation of :math:`V_{cmax}` but also other temperature dependent
+    enzymatic processes.
+
+    The function can operate in one of two modes ("M2002" or "J1942") using the two
+    alternative derivations of the modified Arrhenius relationship presented in
+    :cite:`murphy:2021a`.
+
+    Args:
+        tk: The temperature at which to calculate the factor (K)
+        Ha: The activation energy of the enzyme (:math:`H_a`)
+        Hd: The deactivation energy of the enzyme (:math:`H_d`)
+        deltaS: The entropy of the process (:math:`\Delta S`)
+        tk_ref: The reference temperature for the process (K)
+        mode: The calculation mode.
+        pmodel_const: Instance of :class:`~pyrealm.constants.pmodel_const.PModelConst`.
+        core_const: Instance of :class:`~pyrealm.constants.core_const.CoreConst`.
+
+    PModel Parameters:
+        To: The standard reference temperature expressed in Kelvin (`T_0`, ``k_To``)
+        R: the universal gas constant (:math:`R`, ``k_R``)
+
+    Returns:
+        Values for :math:`f`
+
+    Examples:
```

Comment:
I'm not sure why the example doesn't use the `calc_modified_arrhenius_factor` function

Reply:
Just hadn't got around to updating docstrings properly 😄